updated containers docs (#31884)
dotnet/sdk: src/Containers/README.md

# `Microsoft.NET.Build.Containers`

This folder contains the source code for .NET app containers (`Microsoft.NET.Build.Containers`).

The documentation on .NET app containers is available in [dotnet/sdk-containers-builds](https://github.com/dotnet/sdk-container-builds/tree/main/docs) repo.

In case you encounter an issue with .NET app containers or would like to submit a feature request, please use [dotnet/sdk-containers-builds](https://github.com/dotnet/sdk-container-builds/issues/new) repo.

> If you would like to contribute by creating a pull request, please refer to [dotnet/sdk contributing guide](https://github.com/dotnet/sdk#how-do-i-engage-and-contribute). Ideally, prior starting the effort find a corresponding issue in [dotnet/sdk-containers-builds](https://github.com/dotnet/sdk-container-builds/issues) repo and let us and others know in a comment. If you plan to address the problem that is not reflected in any issue, please create one. Consider helping us triaging the pull request by adding 'Area-Containers' repo.

Reviewer: by adding 'Area-Containers' how exactly?

> This folder contains the source code for .NET app containers (`Microsoft.NET.Build.Containers`).
> 
> The documentation on .NET app containers is available in [dotnet/sdk-containers-builds](https://github.com/dotnet/sdk-container-builds/tree/main/docs) repo.
> 
> In case you encounter an issue with .NET app containers or would like to submit a feature request, please use [dotnet/sdk-containers-builds](https://github.com/dotnet/sdk-container-builds/issues/new) repo.
> 
> If you would like to contribute by creating a pull request, please refer to [dotnet/sdk contributing guide](https://github.com/dotnet/sdk#how-do-i-engage-and-contribute). Ideally, prior starting the effort find a corresponding issue in [dotnet/sdk-containers-builds](https://github.com/dotnet/sdk-container-builds/issues) repo and let us and others know in a comment. If you plan to address the problem that is not reflected in any issue, please create one. Consider helping us triaging the pull request by adding 'Area-Containers' label.

Reviewer 2: ```suggestion
This folder contains the source code for the .NET SDK containers build tools (`Microsoft.NET.Build.Containers`).

Documentation for the tools is available in the [dotnet/sdk-containers-builds](https://github.com/dotnet/sdk-container-builds/tree/main/docs) repo.

If you encounter an issue with the container build tools or would like to submit a feature request, please use the [dotnet/sdk-containers-builds](https://github.com/dotnet/sdk-container-builds/issues/new) repo.

If you would like to contribute by creating a pull request, please refer to [dotnet/sdk contributing guide](https://github.com/dotnet/sdk#how-do-i-engage-and-contribute). Ideally, before starting the effort find a corresponding issue in the [dotnet/sdk-containers-builds](https://github.com/dotnet/sdk-container-builds/issues) repo and let us and others know in a comment. If you plan to address the problem that is not reflected in any issue, please create one. Consider helping us triage the pull request by adding the 'Area-Containers' label.
```
